packages/runtime/src/components/shared/Link/index.tsx

```diff
@@ -147,9 +146,7 @@ export const Link = forwardRef<HTMLAnchorElement, Props>(function Link(
         target={target}
         onClick={handleClick}
         href={href}
-        {...(hasLocalizedPathname && {
-          locale: false,
-        })}
+        locale={false}
```

Comment:
We don't need extra "help" from Next.js in handling localized links.

Reply:
Note that this doesn't affect App Router (I triple checked), which doesn't offer built-in support for localization.

Reply:
Also, as I mention in the [linked issue](https://linear.app/makeswift/issue/ENG-6510/pages-router-links-from-a-localized-page-to-base-pages-dont-work), links work as expected a Pages Router site is using domain-based localization (before and after the fix).